CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.3.9
+- Added types to `toMap` functions
```

Comment:
Please use the same reference pattern as prior changes that links to PR and author.
Please also bump the version in `pubspec.yaml`.